src/Chartjs/Options/Layout.elm

```diff
@@ -1,6 +1,19 @@
 module Chartjs.Options.Layout exposing (Layout(..))
```

Comment:
Is there a better way to handle this file? This type only seems to be used in one spot